src/content/docs/setup/seo/indexing.mdx

```diff
@@ -55,41 +55,26 @@ For [multi-store setups](/merchants/get-started/multistore) or stores supporting
   This is required because Google caches responses without considering headers. Not adding the store code to the URL might lead to Google indexing the wrong data. You can ensure this by adding a cache-busting query parameter.
   :::
 
-#### Cache-busting query parameter
-
-The [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce/tree/main) provides a reference implementation that you can use to add a cache-busting query parameter to the Catalog Service endpoint:
-
-<Tabs>
-  <TabItem label="scripts/commerce.js">
-
-  This function sets the Catalog Service endpoint with a cache-busting query parameter based on the `Magento-Store-Code` header value. The PDP drop-in component uses this function at initialization.
-
-  ```javascript
-  ...
-  export async function commerceEndpointWithQueryParams() {
-    const urlWithQueryParams = new URL(getConfigValue('commerce-endpoint'));
-    // Set some query parameters for use as a cache-buster. No other purpose.
-    urlWithQueryParams.searchParams.append('ac-storecode', getConfigValue('headers.cs.Magento-Store-Code'));
-    return urlWithQueryParams;
-  } 
-  ...
-  ```
-
-  </TabItem>
-  <TabItem label="scripts/initializers/pdp.js">
-
-  This function calls the `commerceEndpointWithQueryParams` function at PDP drop-in component initialization.
-
-  ```javascript
-  ...
-  await initializeDropin(async () => {
-    // Set Fetch Endpoint (Service)
-    setEndpoint(await commerceEndpointWithQueryParams());
-  });
-  ...
-  ```
-  </TabItem>
-</Tabs>
+### Cache-busting Query Parameter
+
+The [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce/tree/main) implements a dynamic cache-busting mechanism for the Catalog Service endpoint. This ensures that when configuration headers change, the browser will fetch fresh data instead of using cached responses.
```

Comment:
```suggestion
The [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce/tree/main) includes a built-in way to prevent stale data from being served. It does this by adding a dynamic cache-busting parameter to Catalog Service requests.

When your configuration headers change, the browser fetches fresh data instead of using cached responses. This helps keep your storefront content up to date.
```